commands/main.go

```diff
@@ -170,6 +171,7 @@ var rootSubcmdsDaemon = map[string]*cmds.Command{
 	"dht":              dhtCmd,
 	"id":               idCmd,
 	"inspect":          inspectCmd,
+	"leb128":           leb128Cmd,
```

Comment:
I have just realised this should be in the "local" list above. It doesn't require a daemon.

Reply:
if need to use local cmd. should merge pr #3382 first.